packages/hardhat-chai-matchers/src/internal/reverted/revertedWithPanic.ts

```diff
@@ -11,6 +12,9 @@ export function supportRevertedWithPanic(Assertion: Chai.AssertionStatic) {
     function (this: any, expectedCodeArg: any) {
       const ethers = require("ethers");
 
+      // see buildAssert's jsdoc
```

Comment:
I'd like to see a one-line explanation here, something like `capture negated flag before async code executes. see buildAssert's jsdoc.`